contracts/Tokens/Prime/PrimeLiquidityProvider.sol

```diff
@@ -95,8 +97,13 @@ contract PrimeLiquidityProvider is
         _;
     }
 
+    /**
+     * @notice Prime Liquidity Provider constructor
+     * @param _blocksPerYear total blocks per year
+     * @param _timeBased A boolean indicating whether the contract is based on time or block.
+     */
     /// @custom:oz-upgrades-unsafe-allow constructor
-    constructor() {
+    constructor(uint256 _blocksPerYear, bool _timeBased) TimeManager(_timeBased, _blocksPerYear) {
```

Comment:
We could define the parameters of the constructor in the same order of the parameters in TimeManager: `bool _timeBased, uint256 _blocksPerYear`